css/styles.css

```diff
@@ -1341,7 +1341,7 @@ ul.video-list {
     overflow: hidden;
 }
 .extra__space{
-    padding-bottom:2.8rem;
+    padding-bottom:1rem;
```

Comment:
Not sure if this is the cause, but it looks like the bottom of the left column's HomeAway item is "overflowing" into the next column or something.

See the top

![screen shot 2018-02-27 at 7 37 37 pm](https://user-images.githubusercontent.com/47366/36768883-0de364e0-1bf6-11e8-988a-da9c44609443.png)

and bottom

![screen shot 2018-02-27 at 7 37 42 pm](https://user-images.githubusercontent.com/47366/36768892-16478d14-1bf6-11e8-81d9-55cd6e74a627.png)

as they show up in my browser.